spec/buftok_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'filewatch/buftok'
+
+describe FileWatch::BufferedTokenizer do
+
+  context "when using the default delimiter" do
+    subject { FileWatch::BufferedTokenizer.new }
+
+    it "splits the lines correctly" do
+      message = "hello\nworld\n"
```

Comment:
you can generalize this with a let statement, probably defined in the first describe area.